backend/import/mo_api.py

```diff
@@ -40,17 +42,23 @@ class MOData:
     """Abstract base class to interface with MO objects."""
 
     def __init__(self, uuid):
+        """Initialize object - speccify session and getter method."""
```

Comment:
```suggestion
        """Initialize object - specify session and getter method."""
```

Neither `session` nor `getter` appear as arguments to this function, which is what the docstring should describe. Perhaps a comment instead? What's `uuid`, though?

Reply:
Fixed.